upstairs/src/lib.rs

```diff
@@ -52,6 +52,12 @@ mod cdt {
     fn gw_read_end(_: u64) {}
     fn gw_write_end(_: u64) {}
     fn gw_flush_end(_: u64) {}
+    fn gw_read_submit_start(_: u64, _: u64) {}
+    fn gw_write_submit_start(_: u64, _: u64) {}
+    fn gw_flush_submit_start(_: u64, _: u64) {}
+    fn gw_read_submit_end(_: u64, _: u64) {}
+    fn gw_write_submit_end(_: u64, _: u64) {}
+    fn gw_flush_submit_end(_: u64, _: u64) {}
```

Comment:
If you're adding or modifying these, now might be a good time to rename them in accordance with some DTrace conventions: dashes between words (rather than snake case) and start/done to bracket an operation.

E.g., rename `gw_read_submit_end` to `gw__read__submit__done`. If you update the `usdt` dependency to the latest (0.3.1), the double-underscores will be converted to dashes in the probe name. So that would become `gw-read-submit-done` at the end of the day.

Reply:
agreed here as well

Reply:
Updated with __ and end->done